kernel/classes/ezcontentobjecttreenode.php

```diff
@@ -4168,12 +4168,10 @@ function subtreeSoleNodeCount( $params = array() )
 
         $nodePath = null;
         $nodeDepth = 0;
-        if ( count( $node ) != 0 )
-        {
+        if (((is_array($node) || $node instanceof Countable) && count($node) != 0) || $node != null) {
```

Comment:
Since we require `symfony/polyfill-php73`, you may simplify these using is_countable(), see https://github.com/ezsystems/ezpublish-legacy/commit/6321a92b4b9a58ef0965415a369dba3a71e79d24#diff-6af1184676109776bea24cee6c02acb7 for more info

Reply:
Thanks for the info, I wasn't aware. Fixed the code to implement `is_countable` instead.